python/lsst/ts/watcher/rules/electric_generator_fail.py

```diff
@@ -0,0 +1,181 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ElectricGeneratorFail"]
+
+import typing
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.xml.enums.Watcher import AlarmSeverity
+
+from ..base_rule import AlarmSeverityReasonType, BaseRule, NoneNoReason
+from ..remote_info import RemoteInfo
+
+
+class ElectricGeneratorFail(BaseRule):
+    """Monitor the electric generators (ESS_agcGenset150)
+    for any alarming states.
```

Comment:
The rule is not monitoring any alarming state but it is monitoring failure states.